vite.config.ts

```diff
@@ -0,0 +1,7 @@
+import { defineConfig } from 'vite'
+import react from '@vitejs/plugin-react'
```

Comment:
**nitpick**: Remove the empty comment line to keep the code clean and maintainable.